internal/indexcoord/flush_segment_watcher.go

```diff
@@ -313,25 +306,24 @@ func (fsw *flushedSegmentWatcher) internalProcess(t *internalTask) {
 		}
 		if err := fsw.writeHandoffSegment(handoffTask); err != nil {
 			log.Error("IndexCoord flushSegmentWatcher writeHandoffSegment with no index fail",
-				zap.Int64("segID", t.segmentInfo.ID), zap.Error(err))
+				zap.Int64("segID", segID), zap.Error(err))
 			return
 		}
 		if err := fsw.removeFlushedSegment(t); err != nil {
```

Comment:
Is it now allowed to delete CompactFrom immediately?

Reply:
SegmentA and SegmentB merged to SegmentC.
if Segment C created index, do we have right to delete Index of SegmentA and SegmentB ?